docs/source/onnxruntime/usage_guides/amdgpu.mdx

```diff
@@ -9,22 +9,31 @@ This guide will show you how to run inference on the `ROCMExecutionProvider` exe
 ## Installation
 The following setup installs the ONNX Runtime support with ROCM Execution Provider with ROCm 5.7. 
 
-#### 1. ROCm Installation
+#### 1 ROCm Installation
 
-To install ROCM 5.7, please follow the [ROCm installation guide](https://rocm.docs.amd.com/en/latest/deploy/linux/index.html).
+Refer to the [ROCm installation guide](https://rocm.docs.amd.com/en/latest/deploy/linux/index.html) to install ROCm 5.7.
 
-#### 2. PyTorch Installation with ROCm Support
+#### 2 Installing `onnxruntime-rocm`
+
+Utilize the provided [Dockerfile](https://github.com/huggingface/optimum-amd/blob/main/docker/onnx-runtime-amd-gpu/Dockerfile) example or perform a local installation from the source since pip wheels are currently unavailable.
```

Comment:
```suggestion
Please use the provided [Dockerfile](https://github.com/huggingface/optimum-amd/blob/main/docker/onnx-runtime-amd-gpu/Dockerfile) example or do a local installation from source since pip wheels are currently unavailable.
```